scripts/x-data-grid-premium.exports.json

```diff
@@ -574,7 +575,6 @@
   { "name": "GridVisibilityOffIcon", "kind": "Variable" },
   { "name": "gridVisibleColumnDefinitionsSelector", "kind": "Variable" },
   { "name": "gridVisibleColumnFieldsSelector", "kind": "Variable" },
-  { "name": "gridVisibleRowsLookupSelector", "kind": "Variable" },
```

Comment:
This feels risky TBH. While `gridVisibleRowsLookupSelector` is not documented, it's exported and not marked as unstable/experimental.
Should we wait with this PR until v7 alpha?